specs/experimental/fault-proof/bond-incentives.md

```diff
@@ -0,0 +1,31 @@
+# Bond Incentives
+
+Bonds is an add-on to the core [Fault Dispute Game](./fault-dispute-game.md). The core game mechanics are
+designed to ensure honesty as the best response to winning subgames. By introducing financial incentives,
+Bonds makes it worthwhile for honest challengers to participate.
```

Comment:
What happens without bonds?

Reply:
Without bonds, honest actors will be guaranteed to win the game. However, it'll be very expensive to do, given the costs of making transactions and running cannon, as they don't get compensated for participating.

Reply:
Added a note on this to the specs Thanks for the review.